reactor-core/src/main/java/reactor/core/publisher/Sinks.java

```diff
@@ -497,6 +502,8 @@ public Emission getReason() {
 	 */
 	public interface Many<T> {
 
+		boolean hasSubscriber();
```

Comment:
Have you considered naming it `hasSubscribers`?

Reply:
actually, since there's no direct need for it in this PR it should probably be debated in a separate issue 🤔

Reply:
Sounds good! Less is better :)